src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderStatic.cs

```diff
@@ -209,6 +218,49 @@ private Vector<T> Load<T>(DataKind kind, int minOrdinal, int? maxOrdinal)
                 return new MyVector<T>(_rec, kind, minOrdinal, maxOrdinal);
             }
 
+            private Key<T> Load<T>(DataKind kind, int ordinal, ulong minKeyValue, ulong? maxKeyValue)
+            {
+                Contracts.CheckParam(ordinal >= 0, nameof(ordinal), "Should be non-negative");
+                Contracts.CheckParam(minKeyValue >= 0, nameof(minKeyValue), "Should be non-negative");
+                Contracts.CheckParam(maxKeyValue == null || maxKeyValue >= minKeyValue, nameof(maxKeyValue), "Should be greater than or eqaul to minimum key value or null");
+                return new MyKey<T>(_rec, kind, ordinal, minKeyValue, maxKeyValue);
+            }
+
+            // A data type used to bridge PipelineColumn and TextLoader's Column. It can be used as PipelineColumn in static-typed
```

Comment:
>// [](start = 12, length = 2)

Turn into summary comment?

Reply:
No problem.

---
In reply to: [229030083](https://github.com/dotnet/machinelearning/pull/1407#discussion_r229030083) [](ancestors = 229030083)